### ISSUE TITLE
Consider switch from private_network to forwarded ports

### ISSUE THREAD

**Issue by enzy** [tsusanka/vagrant-lamp-server#2]
Two network adapters are not necessary and creates a mess in system network adapters (at least on Windows hosts).

**Comment by tsusanka** [comment]
`/etc/hosts` can't [work](https://stackoverflow.com/questions/10729034/can-i-map-a-hostname-and-a-port-with-etc-hosts) with ports - am I wrong?

**Comment by enzy** [comment]
Yep, but it shouldn't be a problem. What difference is `localhost:8000` vs. `v.l` ?

**Comment by tsusanka** [comment]
`v.l` is way more funky. I'll think about it I think I ran to some problems using the ports
